conf/evolutions/default/107.sql

```diff
@@ -0,0 +1,30 @@
+# --- !Ups
+
+INSERT INTO `vitals` (`id`, `name`, `data_type`, `unit_of_measurement`)
```

Comment:
Since there's no database structure being changed i would propose adding these during startup via a new section in the DatabaseSeeder instead:
https://github.com/FEMR/femr/blob/master/app/femr/util/startup/DatabaseSeeder.java

The reason for this is if you:
1) run the Ups evolution on a system and the vitals are placed into the table
2) a user creates entries in the patient_encounter_vitals table that links to one of the vitals in (1)
3) run the Downs evolutions - the Downs will fail because you will be breaking the foreign key constraint between patient_encounter_vitals and vitals

Reply:
Made the changes mentioned above here during testing:

https://github.com/kevinzurek/femr/commit/b0c0bf5b01c7bdd4b42c3d52bfd645a842d78316